src/queries/users/updateUser.js

```diff
@@ -0,0 +1,17 @@
+import db from '../../models';
+
+/**
+* @param {object} what what to update
+ * @param {object} condition where to update
+ * @returns {boolean} return true when successfuly updated otherwise false
+ */
+export default async (what = {}, condition = {}) => {
```

Comment:
why is the variable name `what`

Reply:
It's what we want to update like a password.

Reply:
give it a more appropriate name. `what` is not properly descriptive.

Reply:
I have renamed the what name to value.